subprojects/snapshots/src/main/java/org/gradle/internal/fingerprint/DirectorySensitivity.java

```diff
@@ -33,7 +33,7 @@
     /**
      * Ignore directories
      */
-    IGNORE_DIRECTORIES(snapshot -> snapshot.getType() != FileType.Directory),
+    IGNORE_DIRECTORIES(snapshot -> snapshot.getType() == FileType.RegularFile),
```

Comment:
Is this a good change? Ignoring directories is better captured by the original for me. What'd be different if we left it as it was?